### ISSUE TITLE
Terraform code to deploy Kestra OSS on Azure AKS

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/deployment-templates#8]
Terraform code needed + test if everything is working when deployed on AKS with the Azure Database for PostgreSQL backend

**Comment by soyasis** [comment]
@anna-geller Kestra OSS is already available for Azure via the Microsoft marketplace:
https://marketplace.microsoft.com/en-us/product/kestra_technologies.kestra-open-source-official

Can we close this issue?

**Comment by anna-geller** [comment]
yes, we can close this one if you wish, but it would still be helpful to publish just the terraform code - it seems we already have that because we probably used that to create that marketplace offering. Why not publish this repository in open source so that everybody can use it directly however they want to? 

just sth useful but if you don't want to manage that and/or don't have capacity, no worries, you can close it.
